### ISSUE TITLE
Uploading a Featured image uploads it twice

### ISSUE THREAD

**Issue by Ipstenu** [WordPress/gutenberg#3392]
## Issue Overview

When you add a featured image via Gutenberg, it uploads twice, leaving you with one attached to the post and one not in the media library.

## Steps to Reproduce (for bugs)
1. Make a new post
2. Add featured image
3. Save the post
4. Look in the media library and see the image TWICE

Tested on Safari on a Mac

## Expected Behavior
The image should only upload once.

## Current Behavior
It uploads twice.

**Comment by jughoor** [comment]
Incase if your using Volley api adding this retry method worked for me.
stringRequest.setRetryPolicy(new DefaultRetryPolicy(DefaultRetryPolicy.DEFAULT_TIMEOUT_MS * 2, 1, DefaultRetryPolicy.DEFAULT_BACKOFF_MULT));

this problem occur due to the api retry mechanism.

**Comment by jorgefilipecosta** [comment]
Hi @Ipstenu, I'm not being able to reproduce this problem, could you please let me know if the problem is still happening?

**Comment by Ipstenu** [comment]
No, but now I'm getting `function (){var t=i.result(this,"urlRoot")||i.result(this.collection,"url")||F();if(this.isNew())return t;var e=this.get(this.idAttribute);return t.replace(/[^\/]$/,"$&/")+encodeURIComponent(e)}` when I try to upload, which I'm tracking down. This is likely a conflict with 'something' but it's happening on my Multisite all the time, even with all plugins disabled.

**Comment by Ipstenu** [comment]
This seems to happen on SOME themes and the only correlation I can find is that if your theme inserts new meta boxes.

The problem is I _can't_ reproduce this because every time I activate a theme with a meta box, the featured image upload breaks with the above error ...

And it does this:

![screen shot 2017-11-22 at 11 14 31 am](https://user-images.githubusercontent.com/807903/33145779-969a3e2a-cf76-11e7-8d40-19cc886ab6b2.jpg)

I grabbed Yoast to test, but all you have to do is grab a theme like Genesis and boom, Gutenberg's blowing up again :(

tl;dr: I can't test it because something broke metaboxes, which breaks featured image uploads...

**Comment by youknowriad** [comment]
I tried with Yoast and it's working as expected, I can't reproduce. Can you point me towards a free theme breaking Gutenberg to test? Or maybe this is fixed right now?

**Comment by Ipstenu** [comment]
This still happens on StudioPress' Genesis themes.

**Comment by youknowriad** [comment]
Can we get a free copy of these themes to debug? Can we reproduce with a free theme?

**Comment by Ipstenu** [comment]
I've opened a support ticket with StudioPress to ask them about that (and linked here)

**Comment by youknowriad** [comment]
Thanks @Ipstenu for the theme but I still can't reproduce either of these two featured image bugs

![kapture 2018-01-05 at 11 56 28](https://user-images.githubusercontent.com/272444/34606423-92a47126-f20f-11e7-9692-70510517d52c.gif)

It might be something else, a plugin you have or something?

**Comment by Ipstenu** [comment]
That's wild... Have you tried it on an existing post? That is write some post stuff first, THEN add the image?

The only thing I can think that's 'different' is that and that I use Multiste, but I repro-d it on a clean single site.

**Comment by youknowriad** [comment]
@Ipstenu Same on an existing post. Are you using the "master" version of Gutenberg or the last released one? There might be some fixes not released yet (I recall a multisite fix)

**Comment by Ipstenu** [comment]
No, I'm using the latest released plugin version.

**Comment by youknowriad** [comment]
Ok, we will release soon and you can test to check if it's still a bug after that.

**Comment by Ipstenu** [comment]
![screen shot 2018-01-27 at 27 jan - 7 54 02 pm](https://user-images.githubusercontent.com/807903/35478804-f0aff69e-039b-11e8-985a-006a9d08504c.jpg)

Now it does it THREE times and one was a renamed image 🤯

**Comment by gziolo** [comment]
It should be fixed with #5432. Can you confirm?
